src/import-page/CourseImportPage.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable max-len */
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Container, Layout,
+} from '@edx/paragon';
+import Cookies from 'universal-cookie';
+import SubHeader from '../generic/sub-header/SubHeader';
+import { updateFileName, updateImportTriggered, updateSavingStatus } from './data/slice';
+import ImportStepper from './import-stepper/ImportStepper';
+import { getImportTriggered, getLoadingStatus, getSavingStatus } from './data/selectors';
+import { LAST_IMPORT_COOKIE_NAME } from './data/constants';
+import ImportSidebar from './import-sidebar/ImportSidebar';
+import FileSection from './file-section/FileSection';
+import messages from './messages';
+import InternetConnectionAlert from '../generic/internet-connection-alert';
+import { RequestStatus } from '../data/constants';
```

Comment:
```suggestion
import Cookies from 'universal-cookie';

import SubHeader from '../generic/sub-header/SubHeader';
import InternetConnectionAlert from '../generic/internet-connection-alert';
import { RequestStatus } from '../data/constants';
import { updateFileName, updateImportTriggered, updateSavingStatus } from './data/slice';
import ImportStepper from './import-stepper/ImportStepper';
import { getImportTriggered, getLoadingStatus, getSavingStatus } from './data/selectors';
import { LAST_IMPORT_COOKIE_NAME } from './data/constants';
import ImportSidebar from './import-sidebar/ImportSidebar';
import FileSection from './file-section/FileSection';
import messages from './messages';
```

Reply:
Why we cant write some rule for linter? It's terrible to do it by hands..